config/settings.yml

```diff
@@ -136,6 +136,15 @@ fossgis_valhalla_url: "https://valhalla1.openstreetmap.de/route"
 #microsoft_auth_secret: ""
 #wikipedia_auth_id: ""
 #wikipedia_auth_secret: ""
+
+# Settings to use osm.org production as identity provider
+# Requires confidential OAuth2 app on osm.org with scope "openid"
+# and callback http(s)://{other site}/auth/openstreetmap/callback
+#openstreetmap_auth_id: ""
+#openstreetmap_auth_secret: ""
```

Comment:
As detailed in openstreetmap/openstreetmap-website#5063, we would configure an OAuth 2 application on openstreetmap.org:

* Permissions: “Sign in using OpenStreetMap” (`openid`)
* Redirect URIs: https://www.openhistoricalmap.org/auth/openstreetmap/callback https://api.openhistoricalmap.org/auth/openstreetmap/callback

The Client ID and Client Secret from that application would go here, except that we actually configure these settings [in ohm-deploy](https://github.com/OpenHistoricalMap/ohm-deploy/blob/05e4409c294b86e9f7128a9ead261f7dce8326a6/images/web/config/settings.yml).

Reply:
Nice , Seems the values 
```
#openstreetmap_auth_id: ""
#openstreetmap_auth_secret: " 
```
can be added in 
 https://github.com/OpenHistoricalMap/ohm-deploy/blob/main/images/web/config/settings.yml, since that is the file that we use for deployment in ohm

Reply:
When you’re ready, @jeffreyameyer or I can create an OSM account with the shared team e-mail address and create the OAuth 2 application, then share the ID and secret with you. I assume it would work similarly to what we did in #610 for JOSM, though @mmd-osm please correct me if I’m wrong about that.

Reply:
From: https://github.com/OpenHistoricalMap/issues/issues/1072#issuecomment-2917290161, we use OpenHistoricalMap account to create the  OAuth 2 application